docs/manual_test_plan.md

```diff
@@ -18,3 +18,67 @@ requests are not turned on in production.
+|3|Press "New travel request" button|You see a form with even info and expenses|
+|4|In the Event Name field, type "Test Event 360"|Below the Event Name field you will see a message saying there appears to be a date in the field|
+|5|In the Location field, type "Princeton, NJ"|The form will show the entered value|
+|6|Select an Event Format|The form shows that you have selected the appropriate Event Format|
+|7|Select a Participation type|The form shows that you have selected the appropriate Participation type|
+|8|In the Purpose field, type "Training"|The form will show the appropriate Purpose|
+|9|In the Event Dates field, type "12/14/2026 - 12/18/2026"|The form will show the selected date range|
+|10|In the Travel Dates field, type "12/13/2026 - 12/19/2026"|The form will show the selected date range|
+|11|In the Notes to Approvers field, Type "This is a note"|The form will show the entered note|
+|12|In the Anticipated Expenses section, enter some expenses|The form will show a total for each line and an overall total|
+|13|Press the Submit Request button|A modal will appear indicating a possible date in the Event Name|
+|14|Press the Continue button|You see "Travel request was successfully created."|
+
+#### Editing a travel request
+
+|Step|Action|Expected result|
+|---|---|---|
+|1|Create a travel request as detailed above|You see "Travel request was successfully created."|
+|2|Press the Edit button at the bottom of the page|You should be taken to a page where the fields are editable|
+|3|Make a few changes|The fields should reflect the new values|
+|4|Add a new note to the Approvers|The fom will show the entered note|
+|5|Add or remove some expenses|The total should update to reflect changes|
+|6|Press the Submit Request button|A modal will appear indicating a possible date in the Event Name if you did not remove the numbers|
+|7|Press the Continue button|You see "Travel request was successfully updated."|
+
+#### Setting up a delegation for testing
+|Step|Action|Expected result|
+|---|---|---|
+|1|Create a travel request as detailed above|You see "Travel request was successfully created."|
+|2|Follow [the instructions](https://github.com/pulibrary/approvals/tree/main?tab=readme-ov-file#testing-user-accounts) to make yourself a delegate for your approver|The rake task should be successful
+|3|Click on you netid at the top right to open the menu and select "My Delegations"|You should see your approver in the list of names|
+|4|Click on your approver's name to act as a delegate for them|You should see a banner at the top of the page saying you are acting as a delegate for the selected person|
+
+
+#### Approving a travel request
+
+|Step|Action|Expected result|
+|---|---|---|
+|1|With the application set to act as a delegate for your approver as detailed above|You should see a banner at the top of the page saying you are acting as a delegate for the selected person|
```

Comment:
Per [this discussion in slack](https://pulibrary.slack.com/archives/C031U4XCUH2/p1709751765821149) delegators cannot approve travel requests. This test is not valid. 
It's better to refer first to the rake task `bundle exec rake approvals:make_me_a_supervisor\[cac9,7\]`and from there list the steps to approve a travel request.